backend/fms_core/migrations/0070_v5_1_0.py

```diff
@@ -0,0 +1,138 @@
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        flanker_3prime_forward, _ = Sequence.objects.get_or_create(value=NO_FLANKERS_INDEX_STRUCTURE["flanker_3prime_forward"],
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        flanker_3prime_reverse, _ = Sequence.objects.get_or_create(value=NO_FLANKERS_INDEX_STRUCTURE["flanker_3prime_reverse"],
+                                                                   defaults={"created_by_id": admin_user.id, "updated_by_id": admin_user.id})
+        index_structure = IndexStructure.objects.create(
+            name=NO_FLANKERS_INDEX_STRUCTURE["name"],
+            flanker_5prime_forward=flanker_5prime_forward,
+            flanker_5prime_reverse=flanker_5prime_reverse,
+            flanker_3prime_forward=flanker_3prime_forward,
+            flanker_3prime_reverse=flanker_3prime_reverse,
+            created_by_id=admin_user.id,
+            updated_by_id=admin_user.id
+        )
+        reversion.add_to_revision(flanker_5prime_forward)
+        reversion.add_to_revision(flanker_5prime_reverse)
+        reversion.add_to_revision(flanker_3prime_forward)
+        reversion.add_to_revision(flanker_3prime_reverse)
+        reversion.add_to_revision(index_structure)
+
+        # Create index
+        index = Index.objects.create(name=UNKNOWN_INDEX_NAME, index_structure=index_structure,
+                                     created_by_id=admin_user.id, updated_by_id=admin_user.id)
+        reversion.add_to_revision(index)
+
+def create_olink_index_set(apps, schema_editor):
+    IndexSet = apps.get_model("fms_core", "IndexSet")
+    IndexBySet = apps.get_model("fms_core", "IndexBySet")
+    Index = apps.get_model("fms_core", "Index")
+
+    OLINK_INDEX_SET = "Olink_Default_Index_Set"
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        reversion.set_comment(f"Create index set '{OLINK_INDEX_SET}' for Olink library types.")
+        reversion.set_user(admin_user)
+
+        # Create set
+        index_set = IndexSet.objects.create(name=OLINK_INDEX_SET, created_by_id=admin_user.id, updated_by_id=admin_user.id)
+        reversion.add_to_revision(index_set)
+
+        # Create IndexBySet
+        index = Index.objects.get(name=UNKNOWN_INDEX_NAME)  # Assuming the index was created in the previous step
+        index_by_set = IndexBySet.objects.create(index=index, index_set=index_set,
+                                                created_by_id=admin_user.id, updated_by_id=admin_user.id)
+        reversion.add_to_revision(index_by_set)
+
+def create_olink_library_types(apps, schema_editor):
+    NEW_LIBRARY_TYPE_NAMES = ["Olink_Explore_HT",
+                              "Olink_Explore_3K",
```

Comment:
Use the full numerical value instead Olink_Explore_3072 . The lab may call it otherwise but this is the specific name.